packages/nx/src/hasher/file-hasher.ts

```diff
@@ -9,6 +9,11 @@ import { NativeFileHasher } from './native-file-hasher';
 
 function createFileHasher(): FileHasherBase {
   try {
+    const dotEnv: typeof import('dotenv') = require('dotenv');
+    try {
+      dotEnv.config();
+    } catch {}
+
```

Comment:
Let's make this change separately... there's more env files we should read from.